composer.json

```diff
@@ -41,7 +18,7 @@
     "wpackagist-plugin/pantheon-advanced-page-cache": "^0.1.4",
     "wpackagist-plugin/wp-cfm": "^1.4.5",
     "wpackagist-theme/twentyseventeen": "^1.1",
-    "pantheon-systems/wordpress": "^4.8",
+    "pantheon-systems/wordpress-composer": "dev-master",
```

Comment:
We also only tag when there is a new WordPress version. The [`4.8` tag](https://github.com/pantheon-systems/WordPress/tree/4.8) is missing `pantheon.upstream.yml`.